cmd/connect.go

```diff
@@ -76,6 +78,15 @@ func (o *ConnectOptions) Complete(f Factory, args []string) (err error) {
 		return errors.New(repo.ErrNoRepo, "no qri repo exists\nhave you run 'qri setup'?")
 	}
 
+	if o.Migrate {
+		// any required migration by default interrupts all commands with a prompt
+		// users setting the migrate flag are indicating we should skip the migration
+		// prompt, which is this only possible prompt in the `connect` command.
+		// setting noPrompt to true prior to calling init will have the effect
+		// of auto-running migrations
+		noPrompt = true
```

Comment:
I'm fine with it for now, however this might interfere with future commands which might want an independent prompt/no-prompt switch, but perfectly fine with it for now.

Reply:
agree. I left a big note for this exact reason